packages/extra/test/typescript_definitions/js-joda-extra-tests.ts

```diff
@@ -1,50 +1,117 @@
+    expectType<boolean>(dayOfMonth.equals(dayOfMonth));
+    expectType<number>(dayOfMonth.get(ChronoField.DAY_OF_MONTH));
+    expectType<number>(dayOfMonth.getLong(ChronoField.DAY_OF_MONTH));
+    expectType<number>(dayOfMonth.value());
+    expectType<number>(dayOfMonth.hashCode());
+    expectType<boolean>(dayOfMonth.isSupported(ChronoField.DAY_OF_MONTH));
+    expectType<boolean>(dayOfMonth.isValidYearMonth(YearMonth.from(localDate)));
+    expectType<IsoChronology | null>(dayOfMonth.query(TemporalQueries.chronology()));
+    expectType<ValueRange>(dayOfMonth.range(ChronoField.DAY_OF_MONTH));
+    expectType<string>(dayOfMonth.toString());
+}
+
+function test_DayOfYear() {
+    const localDate = LocalDate.now();
+
+    expectType<DayOfYear>(DayOfYear.from(localDate));
+    expectType<DayOfYear>(DayOfYear.now());
+    expectType<DayOfYear>(DayOfYear.of(1));
+
+    const dayOfYear = DayOfYear.of(28);
+
+    expectType<Temporal>(dayOfYear.adjustInto(localDate));
+    expectType<LocalDate>(dayOfYear.atYear(Year.from(localDate)));
+    expectType<LocalDate>(dayOfYear.atYear(localDate.year()));
+    expectType<number>(dayOfYear.compareTo(dayOfYear));
+    expectType<boolean>(dayOfYear.equals(dayOfYear));
+    expectType<number>(dayOfYear.get(ChronoField.DAY_OF_YEAR));
+    expectType<number>(dayOfYear.getLong(ChronoField.DAY_OF_YEAR));
+    expectType<number>(dayOfYear.value());
+    expectType<number>(dayOfYear.hashCode());
+    expectType<boolean>(dayOfYear.isSupported(ChronoField.DAY_OF_YEAR));
+    expectType<boolean>(dayOfYear.isValidYear(localDate.year()));
+    expectType<IsoChronology | null>(dayOfYear.query(TemporalQueries.chronology()));
+    expectType<ValueRange>(dayOfYear.range(ChronoField.DAY_OF_YEAR));
+    expectType<string>(dayOfYear.toString());
+}
+
 function test_Interval() {
-  const instant = Instant.now();
-  const duration = Duration.ofHours(1);
-
-  Interval.of(instant, duration);
-
-  Interval.parse('2019-08-09T20:38:43.298Z/2019-08-09T20:38:43.298Z');
-
-  const interval = Interval.of(instant, instant);
-  interval.start();
-  interval.end();
-  interval.isEmpty();
-  interval.isUnboundedStart();
-  interval.isUnboundedEnd();
-  interval.withStart(instant);
-  interval.withEnd(instant);
-  interval.contains(instant);
-  interval.encloses(interval);
-  interval.abuts(interval);
-  interval.isConnected(interval);
-  interval.overlaps(interval);
-  interval.intersection(interval);
-  interval.union(interval);
-  interval.span(interval);
-  interval.isAfter(interval);
-  interval.isAfter(instant);
-  interval.isBefore(interval);
-  interval.isBefore(instant);
-  interval.isAfterInstant(instant);
-  interval.isBeforeInstant(instant);
-  interval.isAfterInterval(interval);
-  interval.isBeforeInterval(interval);
-  interval.toDuration();
-  interval.equals({});
-  interval.hashCode();
-  interval.toString();
-
-  LocalDate.ofInstant(interval.end());
+    const instant = Instant.now();
```

Comment:
I can restore original formatting, but then it will become inconsistent.

Reply:
What i meant is that for some time (if reformatting commit goes after DayOf-related changes), js-joda-extra-tests.ts would contain inconsistent indentation.

I can also split reformatting into separate commit in a way that it "predates" DayOf-related changes, but this would require rebase and force-push, which I tried to avoid as this could make your work harder (you would have to start over review).

It's up to you, just let me know.

Reply:
yeah.. i think you are right .. sounds like too much effort for this small of a change ... i just like to have separate commits just for reformatting and then the actual "content" commit ... but i'd say leave it as it is